bin/docker_build.sh

```diff
@@ -1,5 +1,4 @@
-#!/bin/sh
-
+#!/usr/bin/env sh
```

Comment:
This change is a relic from an old commit where I attempt to move towards `bash` then we had a zoom discussion about `bash` vs `sh` and we agreed it would be best to not change until we have a team discussion. I will revert.

- [ ] revert